app/serializers/api/v2/measures/overview_measure_serializer.rb

```diff
@@ -13,6 +13,7 @@ class OverviewMeasureSerializer
 
         has_one :duty_expression, serializer: Api::V2::Measures::DutyExpressionSerializer
         has_one :measure_type, serializer: Api::V2::Measures::MeasureTypeSerializer
+        has_one :additional_code, if: proc { |measure| measure.additional_code.present? }, serializer: Api::V2::AdditionalCodeSerializer
```

Comment:
I don't think we need this if proc. Isn't this the default behaviour?

Reply:
We should add a test for this serializer, too

Reply:
So I think what's happening here is you've been trying to serialize a hashie mash which will ask for the id value to check the presence of the relation. Since you're conditionally serialize two different structures you're getting errors when using this serializer in the case where tehre isn't an additional code